src/Orckestra.Composer.Cart/Services/CartService.cs

```diff
@@ -108,12 +111,14 @@ public async virtual Task<CartViewModel> AddLineItemAsync(AddLineItemParam param
             if (param.Quantity < 1) { throw new ArgumentOutOfRangeException(nameof(param), param.Quantity, GetMessageOfZeroNegative(nameof(param.Quantity))); }
             if (string.IsNullOrWhiteSpace(param.BaseUrl)) { throw new ArgumentException(GetMessageOfNullWhiteSpace(nameof(param.BaseUrl)), nameof(param)); }
 
+            param.CartType = GetCurrentCartType();
```

Comment:
I am not sure we need override here param values, but pass here correct values from places where it used.
![image](https://user-images.githubusercontent.com/6649972/151120962-fcb6b45f-af0b-407f-8c70-f3a294099828.png)
So here you need check ComposerContext if is editing and specify correct CartName and CartType
CartName = ComposerContext.EditingOrderId